lib/service/dataLink.js

```diff
@@ -199,7 +199,7 @@ function process(task, modules, callback) {
   }
 
   const operationHandler = serviceObjectToProcess.resolve(dataOp);
-  return operationHandler(task.request, dataLinkCompletionHandler);
+  return operationHandler(task.request, dataLinkCompletionHandler, modules);
```

Comment:
Need a test which ensures that the modules are actually available inside each `operationHandler`